che--typescript/build.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Some could volume mount a local folder to container .ssh so that they could use .ssh/config file unless you think that it wouldn't be used. I think the proxy part of the config file might.